airflow/executors/celery_executor.py

```diff
@@ -337,6 +343,8 @@ def sync(self) -> None:
 
         if self.adopted_task_timeouts:
             self._check_for_stalled_adopted_tasks()
+        if time.time() - self.stuck_tasks_last_check_time > self.stuck_queued_task_check_interval:
```

Comment:
Won't this keep getting called every sync cycle? Shouldn't `stuck_tasks_last_check_time` be updated to the current time here.

Reply:
Good call